internal/neofs/tree.go

```diff
@@ -33,21 +37,28 @@ type (
 const (
 	versioningEnabledKV = "versioning_enabled"
 	lockConfigurationKV = "lock_configuration"
-	oidKv               = "OID"
+	oidKV               = "OID"
 	fileNameKV          = "FileName"
 	systemNameKV        = "SystemName"
+	isUnversionedKV     = "IsUnversioned"
+	isDeleteMarkerKV    = "IdDeleteMarker"
 
 	settingsFileName  = "bucket-settings"
 	notifConfFileName = "bucket-notifications"
 	corsFilename      = "bucket-cors"
 
-	// bucketSystemObjectsTreeID -- ID of a tree with system objects for bucket
-	// i.e. bucket settings with versioning and lock configuration, cors, notifications
-	bucketSystemObjectsTreeID = "system-bucket"
+	// versionTree -- ID of a tree with object versions.
+	versionTree = "version"
+
+	// systemTree -- ID of a tree with system objects
+	// i.e. bucket settings with versioning and lock configuration, cors, notifications.
+	systemTree = "system"
+
+	separator = "/"
```

Comment:
This separator is hardcoded but eventually it should be `delimiter` parameter of S3 requests?

Reply:
I'm not sure. Otherwise we will have more than path in the tree for the same object. (We discussed this problem [here](https://github.com/nspcc-dev/neofs-s3-gw/issues/416) as I remember)